src/components/RecentSearches/RecentSearches-style.ts

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import styled from "styled-components";
+import { colors, cardrRadius, cardShadow } from "../../Utils/Constances";
+
+export const RecentSearchesContainer = styled.div`
```

Comment:
Add overflow: scroll to handle items amount that exceeds the height of the container